dev.yml

```diff
@@ -15,3 +15,8 @@ test:
     else
       rake test $@
     fi
+
+commands:
```

Comment:
While this is appreciated it has no context in this PR. We can keep it in this I dont need to block for this but please keep the PR focused especially when the changeset is so large

Reply:
Of course, sorry about that. We have a lot of this kind of stuff from when we setup our own CI/development setup so thats our bad. If theres anything small like this that you really dont like, I'd be happy to clean it up for you